server/data/hmppsAuthClient.ts

```diff
@@ -24,58 +16,16 @@ export type AuthUserRole = {
   roleCode: string
 }
 
-function getSystemClientTokenFromHmppsAuth(username?: string): Promise<superagent.Response> {
-  const clientToken = generateOauthClientToken(
-    config.apis.hmppsAuth.systemClientId,
-    config.apis.hmppsAuth.systemClientSecret
-  )
-
-  const authRequest = username
-    ? querystring.stringify({ grant_type: 'client_credentials', username })
-    : querystring.stringify({ grant_type: 'client_credentials' })
-
```

Comment:
This is an important bit when calling the community-api - no username in system token. Hopefully its catered for in the changes below.. I'll check.

Reply:
Now moved to `tokenStore.test.ts` where token is generated based on whether username is supplied